Makefile

```
@@ -22,6 +22,20 @@ lint:
 ######################
 #   docker compose   #
 ######################
+spark:
+	docker compose -p spark -f docker-compose-spark.yml up -d
+
+spark-clean:
+	docker compose -p spark down -v
+
+delta-lake:
+	bash docker-compose-delta-lake.sh
+	docker compose -p delta-lake -f docker-compose-delta-lake.yml up -d
+
+delta-lake-clean:
+	docker compose -p delta-lake down -v
+	rm -rf docker-compose-delta-lake.yml
```

```suggestion
delta:
	bash docker-compose-delta.sh
	docker compose -p delta -f docker-compose-delta.yml up -d

delta-clean:
	docker compose -p delta down -v
	rm -rf docker-compose-delta.yml
```

짧게 delta는 어떠신지 여쭤봅니다! 
make 명령어 사용할 때는 확실히 짧은게 좋더라구요..
의견 주세용~

---

네 반영했습니다~